### DISCUSSION TITLE
A student from a non native english speaking country

### DISCUSSION THREAD

**Discussion by adarshkunwar** [community/community#48975] [GitHub Education]
Hi, 
I am a student from Nepal.
My school hasnt provided me with an education email. 
I do have an ID card, but in it the valid date is marked in my country's native calendar format(BS) rather than the common format (AD).
I was wondering if there is a way for me to clear this , since BS is really ahead of AD. (its 2079 BS right now for instance)

I am a student under a Tribhuwan University( TU) affiliated college (Butwal Kalika Campus) and am currently studying BCA. 3rd sem.

P.S : this is not a troll, BS stands for Bikram Sambat,

**Comment by mateusabelli** [comment]
Hello there, I think you should delete this question and write it again without your id picture, you might be exposing sensitive data to the public.

You can also reach GH through email and send it there, they usually respond within a couple of days.

**Reply by adarshkunwar** [reply]
thanks for letting me know that. I dont have the email of github and its kinda hard to me to get a valid result on google search. 
Do you have that email?

**Reply by mateusabelli** [reply]
Hello, you can send a direct message here https://support.github.com/contact/education

**Reply by samus-aran** [reply]
Appreciate the reply @mateusabelli and highlighting the importance of personal information 👍 

Just to clarify that for these types of inquiries @adarshkunwar that we would not direct users to support. We've recently made[ changes to Education support](https://github.com/orgs/community/discussions/49366) that will auto-close users seeking individual reviews of their application. But we're here to help on the Community 👋

**Comment by samus-aran** [comment]
Hi @adarshkunwar 👋 

We understand that not all students have academic emails you can use your personal one if the school allows it. If your original document is not in English then we advise you to take a photo of the original next to an English translation and gregorian date format.

Hope this helps 👍 Let us know how you get on.
